pallets/gear-builtin/src/lib.rs

```diff
@@ -235,10 +242,17 @@ impl<T: Config> BuiltinDispatcher for BuiltinRegistry<T> {
         let actor_id = dispatch.destination();
 
         if dispatch.kind() != DispatchKind::Handle {
-            unreachable!("Only handle dispatches can end up here");
+            let err_msg = "BuiltinRegistry::run: Only handle dispatches can end up here";
```

Comment:
also add what kind dispatch we got actually

Reply:
Added